schema/deploy/shipping/views.sql

```diff
@@ -3474,6 +3475,34 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         and (latest_invite_date is null or latest_invite_date < current_date - interval '3 days')
     ),
 
+        -- Select enrollments for surveillance testing
+    surveillance as (
+        select
+            redcap_url,
+            redcap_project_id,
+            redcap_record_id,
+            redcap_event_name,
+            redcap_repeat_instance,
+            encountered,
+            individual,
+            latest_invite_date,
+            latest_collection_date,
+            5 as priority,
+            'surveillance' as priority_reason,
+
+            latest_positive_hcov19_collection_date,
+            latest_prior_test_positive_date,
+            prior_test_positive_date_base,
+            alerts_off
+        from uw_enrollments
+        -- Filter to participants who come to campus
+        where (on_campus_freq != 'not_on_campus')
```

Comment:
Should this include `AND on_campus_freq IS NOT NULL`?

Reply:
It should already be excluding rows where `on_campus_freq` is NULL, because the `'somestring' != NULL` (and for that matter `'somestring' = NULL` will both evaluate to null. I confirmed with Jess yesterday thats what she wants. 

If you think it helps readability, I can add that clause to make it more explicit.

Reply:
Ok, that makes sense, no need to add more code if they're already excluded.